### PR TITLE
feat: Adding `<Logo/>` and `<Tags/>` components with updated colors too

### PR DESCRIPTION
**Pull request by rogeruiz**
- [x] **feat: Refactoring Directory/Logo into component**
- [x] **fix: Updating colors**
- [x] **feat: Adding new Directory/Tags component**

### CODE REVIEW

**Review by jvmontes — CHANGES_REQUESTED** [review]
let's get aligned with the colors, but i love what u did with the Logo and Tags components!

File: src/app/globals.css [hunk 1]
```diff
@@ -61,6 +66,11 @@
     --sidebar-accent-foreground: 240 4.8% 95.9%;
     --sidebar-border: 240 3.7% 15.9%;
     --sidebar-ring: 217.2 91.2% 59.8%;
+
+    --brand-color: #0a131f;
+    --primary-color: #315dc4;
+    --secondary-color: #367dbf;
+    --accent-color: #dba41a;
```
[the remark this answers is not in the document]
Comment (rogeruiz) [reply src/app/globals.css:73]: I used a tool called Coolers.co which allows me to shade colors down to get a dark mode equivalent

File: src/app/globals.css [hunk 2]
```diff
@@ -30,6 +30,11 @@
   --sidebar-ring: 217.2 91.2% 59.8%;
 
   --radius: 0.5rem;
+
+  --brand-color: #12263a;
+  --primary-color: #2d56b2;
+  --secondary-color: #8fb8de;
+  --accent-color: #f0d07d;
```
[the remark this answers is not in the document]
Comment (rogeruiz) [reply src/app/globals.css:37]: I can change those and the dark colors into HSL later today.

[the remark this answers is not in the document]
Comment (rogeruiz) [reply src/app/globals.css:37]: Also CSS variables can hold arbitrary strings, I'd like for the name of the variable to hold the type of value it is. But we don't need to include the word color in the Tailwind configuration.

**Comment by rogeruiz** [comment]
@jvmontes Okay I've updated all the colors to use `hsl()` and moved the call to that function in to the variable declaration rather than where we're referencing the variable. This way we can modify the colors from `hsl()` if we need to. If there's more feedback about colors, I'd feel better to change things in other PRs rather than in this PR. I still need this to be stable before I can get started on the Org Details page again.

File: tailwind.config.ts [hunk 3]
```diff
@@ -40,52 +40,58 @@ const config: Config = {
...
       borderRadius: {
         lg: "var(--radius)",
         md: "calc(var(--radius) - 2px)",
         sm: "calc(var(--radius) - 4px)",
+        cardHover: "var(--card-hover)",
+
+        brand: "var(--brand)",
+        accent: "var(--accent)",
+        primary: "var(--primary)",
+        // secondary: "var(--secondary)",
```
Comment (jvmontes) [inline tailwind.config.ts:94]: nitpick but we could probably uncomment this even if it isn't used.

**Review by jvmontes — APPROVED** [review]
Looks good! Approved 👍🏼

**Pull request merged**